types/src/moana_types.cr

```diff
@@ -109,6 +109,8 @@ module MoanaTypes
       pool = Pool.new,
       distribute_groups = [] of DistributeGroup,
       no_start = false,
+      reuse_storage_units = false,
+      volume_id = "",
```

Comment:
id is already there. `volume_id` not required.

Reply:
removed now.